ui/src/views/Connectors.vue

```diff
@@ -680,8 +680,9 @@ export default {
     },
     openEditConnector(conn, email) {
       this.newConnector = Object.assign({}, conn);
-      this.newConnector.props.FilterCheck =
-        this.newConnector.props.FilterCheck == "enabled";
+      this.newConnector.props.FilterCheck == "enabled"
```

Comment:
You can remove the string compare with `enabled` because the checkbox behavior in modal is correct both with `enable` or `disabled`

So remove the lines: 683, 684, 685

Reply:
Removed.